oh.sh

```diff
@@ -49,8 +49,8 @@ WRITE_CONFIG_FILES="off"
 #OH_MODE="PORTABLE" 
 
 # language setting - default set to en
-OH_LANGUAGE_LIST="en|fr|es|it|pt|ar"
-#OH_LANGUAGE="en" # default
+OH_LANGUAGE_LIST=("al" "ar" "de" "en" "es" "fr" "it" "pt")
```

Comment:
```suggestion
OH_LANGUAGE_LIST=("ar" "de" "en" "es" "fr" "it" "pt" "al")
```

or (to not switch INFO)

```suggestion
OH_LANGUAGE_LIST=("sq" "ar" "de" "en" "es" "fr" "it" "pt")
```

Reply:
fixed, thanx